convert-stage-4.sh

```diff
@@ -23,6 +23,7 @@ Usage: $0 [options]
         --server-cert       - Mender server certificate
         --tenant-token      - Mender tenant token
         --keep              - Keep intermediate files in output directory
+        --no_service        - Don't enable mender service during boot
```

Comment:
Can we name the parameter `--no-service`? Using dash instead of underscore is a more extended practice (and already in use in the current script